lms/static/coffee/spec/instructor_dashboard/student_admin_spec.coffee

```diff
@@ -0,0 +1,166 @@
+describe 'StudentAdmin', ->
```

Comment:
@ziafazal Sorry but I'm still confused. Whether the code being tested is in CoffeeScript or JavaScript doesn't affect the language you choose to write the tests in. There should not be natively written JS files under the coffee directory as that is confusing. People expect all JS files under those directories to have been generated and so think that they can safely be deleted. 

I think the piece that I missed is that you aren't using RequireJS yet. I would suggest that you change your file to use it to declare the dependencies, and you can then register the file itself in the main.js file as I suggested before. In addition, you can declare any dependencies that you need as shims in this file, so that they will get loaded before the test is run.

I think that ecommerce_spec.js is still the best file for you to model your tests on. This is an instructor dashboard test that is written in JS using our current best practices.